ui/src/components/error-boundary/error-boundary.tsx

```diff
@@ -0,0 +1,39 @@
+import { Button } from 'design-system/components/button/button'
+import { Icon, IconTheme, IconType } from 'design-system/components/icon/icon'
+import { Tooltip } from 'design-system/components/tooltip/tooltip'
+import { ErrorInfo, ReactNode } from 'react'
+import { ErrorBoundary as _ErrorBoundary } from 'react-error-boundary'
+import { STRING, translate } from 'utils/language'
+import styles from './error-boundary.module.scss'
+
+const logErrorToService = (error: Error, info: ErrorInfo) => {
+  // TODO: Pass error to Sentry here
```

Comment:
@mihow here is where we would log errors to Sentry, so that you and me can get info about what happened and get a chance to make a proper fix :) Info objects should be pretty detailed

Reply:
Oh that's great! Check for an invitation from Sentry. I just sent you a new one from our sponsored account.

Reply:
Thanks! Sorry I forgot the last invite. FE app is now hooked up with Sentry 🚀

<img width="1625" alt="Screenshot 2023-11-23 at 09 56 42" src="https://github.com/RolnickLab/ami-platform/assets/11680517/9867039d-43a8-485b-830a-1d2ab096ed92">